qpc/messages.py

```diff
@@ -233,7 +234,9 @@
 BECOME_PASSWORD = 'Provide a privilege escalation password to be used when '\
     'running a network scan.'
 
-
+JOB_ID_NOT_FOUND = 'Merge report job "%s" not found.'
+JOB_ID_STATUS = 'Job id: %s status is %s'
+DISPLAY_REPORT_ID = 'qpc report summary --report "%s" --csv.'
```

Comment:
Add to prefix so it says 'To view report, run: qpc report summary --report "%s" --csv.'